index.html

```diff
@@ -233,5 +233,78 @@ <h1>About me</h1>
 				</ul>
 			</div>
 		</section>
+		<section id="form-section">
+			<div id="form-text" class="form-text">
+				<p class="text">
+					I'm always interested in hearing about new projects, so if you'd like
+					to chat please get in touch.
+				</p>
+			</div>
+			<form
+				action="https://formspree.io/f/form"
+				id="form"
```

Comment:
I can see you are following the structure of the form action URL here, 👍.

- [ ] Your form is not being submitted right now due to the wrong action URL given. Kindly create an account on the form spree website, then create a form on the account you created and copy the action URL of that account. It's that URL that you need to pass on the form's action attribute in your project. Please fix that 👍
- Below is the response from form spree
![image](https://user-images.githubusercontent.com/50941074/166830303-e4fbe241-064a-49e4-ac06-a62983a41fef.png)